sql_to_code/parsers/alter_table/parser.py

```diff
@@ -1,32 +1,33 @@
-import re
+from pyparsing import *
 
 from .models import ForeignKey, Reference
 
-source_table_name_regex = re.compile('ALTER TABLE "(?P<table_name>\w+)"')
-foreign_key_name_regex = re.compile('ADD FOREIGN KEY \("(?P<foreign_key>\w+)"\)')
-result_table_name_regex = re.compile('REFERENCES "(?P<result_table_name>\w+)"')
-result_table_field_name_regex = re.compile(
-    'REFERENCES "\w+" \("(?P<result_table_field_name>\w+)"\);'
+table_name_schema = CaselessKeyword("alter table") + QuotedString('"')("table_name")
```

Comment:
For example, talking about first comment, looks there is [quotedString variable pre-defined](https://github.com/pyparsing/pyparsing/blob/master/pyparsing/core.py#L4834) which already getting into the account possible types of quotes. Both of them (QuotedString and this quotedString variable result) are inherited from [one class](https://github.com/pyparsing/pyparsing/blob/f10fcf8c98ea07a63f8286288ce82f773e7e152e/pyparsing/core.py#L3701) so it is the same ball game

Reply:
what i wanted to do as a first step is get rid of direct usage of regex expressions in our code, so improvements for parser schemas can be done in separated pull request

Reply:
Okay, I was wrong about quotes and in case dbs' identifiers it will be always double quotes (at least it should be according to identifier) or **not quoted at all** (still should be done). Nothing to solve here